pyvista/utilities/common.py

```diff
@@ -0,0 +1,47 @@
+"""Common functions."""
+import collections.abc
+
+import numpy as np
+
+from pyvista._typing import PointsLike
+
+
+def _coerce_pointslike_arg(points: PointsLike,
```

Comment:
`PointsLike` remnants here tripping CI. But github's having major issues with PRs and actions so don't be surprised if there's a surprising failure after fixing this.

Reply:
This is my fault.  I must have forgotten to `git add` all the right files.  I had taken your advice on using a precommit git hook, but it doesn't prevent oversights like this.  It is possible that the python pre-commit package does this.  I'm going to look into it.

Reply:
Well there's always `git commit -a`, but I personally recommend against using it. But that's mostly because I love being explicit and it's annoying to clean up after accidental eager `add`s. I just do a `git status` before or after committing and check that everything is green rather than red in the output.